doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -107,25 +90,23 @@ spec:
 EOF
 ```
 
-Then wait for the `ClusterIssuer` to become ready as follows:
+Wait for the `ClusterIssuer` to become ready.
 
 ```bash
-kubectl wait clusterissuer/${clusterIssuerName} --for=condition=ready=true
+kubectl wait clusterissuer/${KUADRANT_CLUSTER_ISSUER_NAME} --for=condition=ready=true
 ```
 
-### Step 4 - Set up a Gateway
-
-For Kuadrant to balance traffic using DNS across two or more clusters, you must define a Gateway with a shared host. You will define this by using an HTTPS listener with a wildcard hostname based on the root domain. As mentioned earlier, you must apply these resources to all clusters. 
+### Deploy a Gateway
 
-NOTE: For now, the Gateway is set to accept an `HTTPRoute` from the same namespace only. This allows you to restrict who can use the Gateway until it is ready for general use.
+For now, the Gateway is set to accept an `HTTPRoute` from the same namespace only. This allows the ability to restrict who can use the Gateway until it is ready for general use.
```

Comment:
This read a little funny, maybe something like:

```suggestion
At present, the Gateway is limited to accepting `HTTPRoute` resources only from its own namespace. This restriction helps control access, ensuring the Gateway remains secure until it is ready for wider use.
```

made it a little less passive voice